.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: '*'
+  pull_request:
+    branches: main
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
+
+      - name: Install behave
+        run: pip3 install -r requirements.txt
+      
+      - name: Run behave tests
+        run: behave
```

Comment:
ha jól értem, ezt a parancssorba adja be lényegében, akkor meg kell adni paraméternek a feature fileokat, pl.: behave binsearch.feature, és ebből van 3 db, binsearch, caesarcipher, bubblesort

Reply:
Ha `behave`-et futtatsz, akkor minden tesztet lefuttat a ./features-ben.